packages/next/package.json

```diff
@@ -289,7 +289,7 @@
     "punycode": "2.1.1",
     "querystring-es3": "0.2.1",
     "raw-body": "2.4.1",
-    "react-is": "19.0.0-canary-94eed63c49-20240425",
+    "react-is": "19.0.0-rc-f994737d14-20240522",
```

Comment:
Can you add logic to `scripts/sync-react.js` to make sure this is always updated?

Reply:
I've just taken a look and don't think I understand the `scripts/sync-react.js` logic enough to add this functionality.

Reply:
I can follow-up on this

Reply:
Will be handled in https://github.com/vercel/next.js/pull/67189/